src/main/java/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -24,13 +25,15 @@
     private final VictorSPX leftBack = new VictorSPX(Ports.leftBackMotor);
     private final VictorSPX rightForward = new VictorSPX(Ports.rightForwardMotor);
     private final VictorSPX rightBack = new VictorSPX(Ports.rightBackMotor);
-
+    public Point robotPoint;

```

Comment:
This name is not so good. What exact point is it?
```suggestion
    public Point currentLocation;
```